OneCommunityGlobal/HighestGoodNetworkApp
# Mounica - Make managers, mentors, and assistant manager obvious on teams.

# Description
Make managers, mentors, and assistant manager obvious on teams. 
1. Order them from most to least permissions: Owner, Admin, Core Team, Manager, Assistant Manager, Mentor… So owner would be at top. 
2. Make them bold and include their role after their name and in parentheses (i.e. Tim Kent (Manager)

## How to test:
1. check into current branch
3. do `npm install` and `...` to run this PR locally
4. log as admin user
5. Go to Other Links>Teams>Choose Team 

## Screenshots or videos of changes:
![Screenshot from 2023-09-06 22-33-44](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/49270410/6b23e32e-b3af-4005-94ce-739b23a14306)


## Note:
Include the information the reviewers need to know.
